roles/openshift_facts/tasks/main.yml

```diff
@@ -26,7 +34,12 @@
       cluster_id: "{{ openshift_cluster_id | default('default') }}"
       hostname: "{{ openshift_hostname | default(None) }}"
       ip: "{{ openshift_ip | default(None) }}"
-      is_containerized: "{{ l_is_containerized | default(None) }}"
+      is_containerized: "{{ l_is_containerized | default(l_is_system_containers) }}"
```

Comment:
l_is_containerized is always going to be set because of the set_fact above, should be able to just use `l_is_containerized or l_is_systemcontainers` here.

Only time default is really needed is if the value might not be set, or if we are trying to make sure it isn't any empty string, list, etc.